src/pjpersist/datamanager.py

```diff
@@ -665,6 +667,15 @@ def _join_txn(self):
             self._needs_to_join = False
             self._begin(transaction)
 
+    def _doom_txn(self):
+        try:
+            self.transaction_manager.doom()
```

Comment:
Shouldn't we doom the transaction instead of transaction manager?

Reply:
I can if you prefer.  It's the same thing.

Documentation says:
```
class ITransactionManager(Interface):
<...>
    def doom():
        """Doom the current transaction.

        In explicit mode, if a transaction hasn't begun, a
        `NoTransaction` exception will be raised.
        """
```

Change it to `self.transaction_manager.get().doom()`, hopefully this is less confusing.